release/release_tests.yaml

```diff
@@ -768,6 +768,7 @@
 - name: shuffle_10gb
   group: Core nightly tests
   team: core
+  frequency: multi
```

Comment:
multi not nightly? microbenchmark is nightly

Reply:
This is for shuffle_10gb! (I realized this test wasn't running without specifying frequency)